website/src/pages/demodays.js

```diff
@@ -1,72 +1,124 @@
-import Layout from '@theme/Layout'
-import yash from '/static/img/demo-days/yashkandalkar.jpg'
-import radis from '/static/img/demo-days/radistoumpalidis.jpg'
-import palak from '/static/img/demo-days/palaksharma.jpg'
-
+import styles from './demodays.module.css';
+import Layout from '@theme/Layout';
+import yash from '../../static/img/demo-days/yashkandalkar.jpg';
+import radis from '../../static/img/demo-days/radistoumpalidis.jpg';
+import palak from '../../static/img/demo-days/palaksharma.jpg';
+import Janvi from '../../static/img/demo-days/janvithakkar.jpg';
+import Sanjay from '../../static/img/demo-days/sanjayrajpoot.jpg';
 
 const Hello = () => {
-    return (
-        <Layout title="Hello" description="Hello React Page">
-            <div>
-                <section className={styles.intro}>
-                    <div>
-                        <h1 className={styles.title}>Demo Days 2022</h1>
-                        <h2 className={styles.date}> 🗓️ Saturday, 22<sup>nd</sup> October</h2>
-                    </div>
-                </section>
-                <section className={styles.about}>
-                    <div className={styles.descript}>
-                        <h1 className={styles.head}>Agenda</h1>
-                        <p className={styles.headscript}>
-                            On October 22, 12:30 PM GMT, we will have our GSoC mentees showcasing their work and learnings to the community
-                        </p>
-                    </div>
-                    <div className={styles.descript2}>
-                        <h1 className={styles.head}>What to Expect</h1>
-                        <ol className={styles.point}>
-                            <li>Get to know the community better</li>
-                            <li>Get familiar with the projects</li>
-                            <li>oppurtunities for new contibutors</li>
-                        </ol>
-                    </div>
-                </section>
+  return (
+    <Layout title="Hello" description="Hello React Page">
+      <div>
+        <section className={styles.intro}>
+          <div>
+            <h1 className={styles.title}>Demo Days 2022</h1>
+            <h2 className={styles.date}>
+              {' '}
+              🗓️ Saturday, 22<sup>nd</sup> October
+            </h2>
+            <h2 className={styles.date}>
+              {' '}
+              🗓️ Saturday, 24<sup>th</sup> December
+            </h2>
+            <h2 className={styles.date}> 🗓️ Coming soon...</h2>
+          </div>
+        </section>
+        <section className={styles.about}>
+          <div className={styles.descript}>
+            <h1 className={styles.head}>Agenda</h1>
+            <p className={styles.headscript}>
+              We will have our Graduted mentees from GSoC(Google summer of code), LFX(Linux
+              Foundation), GSOD(Google season of docs) and Outreachy from the batch 2022 showcasing
+              their work and learnings with the community.
```

Comment:
- Change to `graduated mentees`
- Keep a space between the terms and their abbreviation